docs/backend-api/backend-api-installation.md

```diff
@@ -0,0 +1,36 @@
+# Backend API Installation
+
+Require `shopsys/backend-api` package in the `composer.json` or run
+
+```bash
+composer require shopsys/backend-api
+```
+
+Backend API installation is fully automated, so you have to run only (you have to be in the project root directory)
+
+```bash
+bash vendor/shopsys/backend-api/install/install.sh
+```
+
+If it was successful, you'll see `Backend API installation was successful!`
+
+The last step is to update your composer because the `install.sh` added a composer dependency
+
+```bash
+composer update
+```
+
```

Comment:
Should be running in docker considered in this guide? Because to me, it was all clear as I run it natively, but when you running in docker, you need to perform these steps inside of a container. 
I'm not really sure how we should document this type of things.

Reply:
And what about db migrations? I need to run them, but neither here, end neither in following article is no information about it